src/components/UTIL_PageMessages.component

```diff
@@ -14,6 +14,13 @@
         required="false"
         />
 
+    <apex:attribute name="escMsg"
+        description="If true, the page message will be escaped, by default it will not."
+        type="Boolean"
+        default="true"
+        required="false"
```

Comment:
your description says by default it will not escape, but your default value = true. shouldn't we make it default=true, since that is the most secure behavior.  those that explicitly want to override escape need to say so (escape=false).  this would be consistent with other visualforce components.

Reply:
Yes, sorry I was going to default to false worried about the other consumers but after checking through the pages that use links they appear to use them directly vs in the error message. I was  unable to find anything that appeared to use links in the errors.

Reply:
Good catch, I was looking at the hrefs we were using in labels and tested the contact delete override but not with child opps.